backend/src/main/java/de/uslu/Travel/TravelController.java

```diff
@@ -15,13 +15,19 @@
     private final TravelService travelService;
 
     @GetMapping
-    public Collection<Travel> listOfTravel(Principal principal) {
-        return travelService.listTravels(principal.getName());
+    public Collection<Travel> listOfTravel(String travel) {
+        return travelService.listTravels(travel);
```

Comment:
```suggestion
    public Collection<Travel> listOfTravel(String city) {
        return travelService.listTravels(city);
```

Since you filter by city